### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by lwrubel** [sul-dlss/was-pywb#100]
Running the automated accessibility tools SiteImprove extension and [Pa11y](https://github.com/pa11y/pa11y) on SWAP yielded a few improvements we could make to `search.html`, `frame_insert.html`, `playback.css`, and possibly `base.html` (all files we are already overriding). 

### Home / search page 
(https://swap.stanford.edu)

SiteImprove:
* 2.5.3 Label in Name _(related to local change, not in upstream pywb)_
Visible label and accessible name do not match
The accessible name of any interactive element should contain its visible text label.
_(refers to Submit button having label "Search". aria-label should be changed to match.)_

Pa11y did not identify any errors. 

### Replay page 
(e.g. https://swap.stanford.edu/was/20220601160941/http://news.stanford.edu/)
This code should be in `frame_insert.html`

From the SiteImprove extension:

* 4.1.2 Name, Role, Value
Inline frame without a text alternative:
Every inline frame (iframe) should have a text alternative that summarizes its content or purpose.
* 3.1.1 Language of Page
Page language has not been identified
The main language of the page has not been declared within the <html> element.
_This is set on the home / search page, but not implemented in frame_insert.html._
* WAI-ARIA authoring practices
Text not included in an ARIA landmark _(refers to text in the replay header, such as the data, previous and next captures, site title)_
All perceivable text content should be included in an [ARIA landmark](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html)
* Accessibility best practices
Page has no headings _(not sure we should do anything to address this, including this in the ticket just for documentation)_
This page does not appear to have any heading tags (h1 through to h6), or any other elements with the role of heading.
 
(There were two other items relating to the content of the replayed page, not addressable through pywb)

From Pa11y:

```
Results for URL: https://swap.stanford.edu/was/20220601160941/http://news.stanford.edu/

 • Error: The html element should have a lang or xml:lang attribute which describes the language of the document.
   ├── WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
   ├── html
   └── <html><head> <meta http-equiv="cont...</html>

 • Error: Iframe element requires a non-empty title attribute that identifies the frame.
   ├── WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
   ├── #replay_iframe
   └── <iframe id="replay_iframe" frameborder="0" seamless="seamless" scrolling="yes" class="wb_iframe" allow="autoplay; fullscreen" src="https://swap.stanford.edu/was/20220601160941mp_/http://news.stanford.edu/"></iframe>
```

### Page with list of capture dates
(e.g. https://swap.stanford.edu/was/*?url=news.stanford.edu)
This code is in the footer and CSS.

From the SiteImprove extension:
* 1.4.8 Visual Presentation  _(related to local change, not in upstream pywb)_
Font size is fixed _(referring to the copyright statement in the footer)_
The font-size has been fixed using an absolute length unit
(https://www.w3.org/TR/css-values-3/#absolute-lengths), such as px, pt or cm.

From Pa11y
```
 > Running Pa11y on URL https://swap.stanford.edu/was/*/http://news.stanford.edu/

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation:  change text colour to #0072ed.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── #year-tab-list > a:nth-child(1)
   └── <a class="list-group-item list-group-item-action active" role="tab" href="#_2010-Display-Month" data-toggle="list">2010</a>

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation:  change text colour to #0072ed.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── #_2010-Month-Display-List > a
   └── <a class="list-group-item list-group-item-action active" role="tab" href="#_2010-August-Display-Days" data-toggle="list">August</a>

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation:  change text colour to #0072ed.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── #_2010-August-Display-Days-List > li > a
   └── <a href="https://swap.stanford.edu/was/20100805003657/http://news.stanford.edu/" target="_blank">August 5th, 2010 at 00:36:57 </a>

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.04:1. Recommendation:  change background to #008297.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── #count_20100805003657
   └── <span id="count_20100805003657" class="badge badge-info badge-pill float-right">1</span>

```

**Comment by lwrubel** [comment]
These accessibility items are a mix of items we introduced and can control, and items in pywb. I'll inquire in the IIPC Slack about the ones that are coming from pywb and create an issue/PR. 

The item about fixed font size in the footer copyright statement is coming from `su-identity.css`.

Adding an ARIA landmark role to the replay header would require overwriting `default_banner.js` which we are particularly trying to avoid.
